graph/src/data/query/error.rs

```diff
@@ -83,6 +83,63 @@ pub enum QueryExecutionError {
     ResultTooBig(usize, usize),
 }
 
+impl QueryExecutionError {
+    pub fn is_attestable(&self) -> bool {
+        use self::QueryExecutionError::*;
+        match self {
+            OperationNameRequired
+            | OperationNotFound(_)
+            | NotSupported(_)
+            | NoRootSubscriptionObjectType
+            | NonNullError(_, _)
```

Comment:
You would expect this to be consistent across indexers but let's say one of them wipes one of the entity tables of a subgraph and suddenly all references that are supposed to be non-null are `null`... then this _could_ be non-deterministic. Therefor I'd move it into the unattestable category.